src/context/WalletProvider/Ledger/components/Chains.tsx

```diff
@@ -123,27 +138,53 @@ export const LedgerChains = () => {
 
   const chainsRows = useMemo(
     () =>
-      availableAssets.map(asset => {
-        return (
-          <Flex alignItems='center' justifyContent='space-between' mb={4} key={asset.assetId}>
-            <Flex alignItems='center' gap={2}>
-              <AssetIcon assetId={asset.assetId} size='sm' />
-              <RawText fontWeight='bold'>{asset.name}</RawText>
-            </Flex>
-            <Button
-              isLoading={loadingChains[asset.chainId]}
-              // we need to pass an arg here, so we need an anonymous function wrapper
-              // eslint-disable-next-line react-memo/require-usememo
-              onClick={() => handleConnectClick(asset.chainId)}
-              colorScheme={walletChainIds.includes(asset.chainId) ? 'green' : 'gray'}
-              isDisabled={walletChainIds.includes(asset.chainId)}
-              leftIcon={walletChainIds.includes(asset.chainId) ? <CheckCircleIcon /> : undefined}
-            >
-              {walletChainIds.includes(asset.chainId) ? 'Added' : 'Connect'}
-            </Button>
+      availableAssets.map(asset => (
+        <Flex alignItems='center' justifyContent='space-between' mb={4} key={asset.assetId}>
+          <Flex alignItems='center' gap={2}>
+            <AssetIcon assetId={asset.assetId} size='sm' />
+            <RawText fontWeight='bold'>{asset.name}</RawText>
+            {asset.assetId === thorchainAssetId && (
+              <Popover trigger='hover' placement='right-start'>
+                <PopoverTrigger>
+                  <Box>
+                    <Icon
+                      as={InfoOutlineIcon}
+                      boxSize={4}
+                      ml={1}
+                      color='gray.500'
+                      cursor='pointer'
+                    />
+                  </Box>
+                </PopoverTrigger>
+                <PopoverContent>
+                  <PopoverArrow />
+                  <PopoverBody>
+                    {'Follow the instructions '}
+                    <Link
+                      color='blue.200'
+                      href='https://support.ledger.com/hc/en-us/articles/4402987997841-THORChain-RUNE-?docs=true'
+                    >
+                      here
+                    </Link>
+                    {' to get the RUNE app on your Ledger'}
```

Comment:
No translation yet, will move all Ledger copies to translation files after a final UX revieew